scripts/push_binary.sh

```diff
@@ -30,5 +30,5 @@ fi
 gcloud builds submit \
     --project=datcom-ci \
     --config=build/ci/cloudbuild.push.yaml \
-    --substitutions SHORT_SHA="$TAG" \
+    --substitutions SHORT_SHA="$TAG",_SKIP_AUTOPUSH_UPDATE=true \
```

Comment:
nit: should an external var not start with `_`?

Reply:
cloudbuild requires custom variable to start with _: https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_user-defined_substitutions